hs_core/irods.py

```diff
@@ -140,7 +148,8 @@ def create_ticket(self, user, path=None, write=False, allowed_uses=1):
         formatted = timeout.strftime("%Y-%m-%d.%H:%M")
         istorage.session.run('iticket', None, 'mod', ticket,
                              'expires', formatted)
-        return ticket
+
+        return ticket, self.irods_full_path(path)
```

Comment:
It seems to me that the line 132 as shown below does the same thing as calling ```self.irods_full_path(path)```, hence my comment above.
```
if path.startswith(self.short_id) or path.startswith('bags/'):  # local path
            path = os.path.join(self.irods_home_path, path)
```